src/main/java/ch/njol/skript/expressions/ExprExcept.java

```diff
@@ -0,0 +1,81 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.skript.util.LiteralUtils;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.lang.comparator.Comparators;
+import org.skriptlang.skript.lang.comparator.Relation;
+
+@Name("Except")
+@Description("Filter a list by providing objects to be excluded.")
+@Example("""
+		spawn zombie at location(0, 0, 0):
+			hide entity from all players except {_player}
+	"""
+)
+@Example("""
+		set {_items::*} to a copper ingot, an iron ingot and a gold ingot
+		set {_except::*} to {_items::*} excluding copper ingot
+	"""
+)
```

Comment:
```suggestion
	""")
@Example("""
		set {_items::*} to a copper ingot, an iron ingot and a gold ingot
		set {_except::*} to {_items::*} excluding copper ingot
	""")
```
that other change request led you astray didn't it lol